layouts/shortcodes/checkin-schedule.html

```diff
@@ -44,11 +44,15 @@
 
     function getDateFromWeek(week, year) {
         var day = (2 + (week - 1) * 7);
-        return new Date(year, 0, day);
+        var d = new Date(year, 0, day, 5);
```

Comment:
here I am adding five hours to stay away from the beginning time of the day (00:00:00)

Reply:
```suggestion
        var d = new Date(Date.UTC(year, 0, day));
```
I already see 2020-10-29 on the website right now, so I have a suspicion that this is related to timezones and that the existing logic is broadly correct - could you check if this change fixes the issue instead?

Reply:
I was going to say exacly what @davidtwco just told you :). Better to use UTC and problem solved because that +5 could lead to issues I guess if I check this at 11pm :).

Reply:
great suggestion @davidtwco I see the same output but it's now safer. Right now the week has just rotated, as expected. Current output:
```
Previous Week: 2020-10-22 - 2020-10-29 Meta and MIR Optimizations
This Week: 2020-10-29 - 2020-11-05 Polonius and Polymorphization
Next Week: 2020-11-05 - 2020-11-12 RFC 2229 and RLS 2.0
```